tracing-journald/src/lib.rs

```diff
@@ -268,6 +298,60 @@ where
     }
 }
 
+/// The facility argument is used to specify what type of program is logging the
+/// message. This lets the configuration file specify that messages from
+/// different facilities will be handled differently.
+#[derive(Copy, Clone)]
+pub enum SyslogFacility {
```

Comment:
do we know that the list of syslog facilities will always be the same across various operating systems and libcs? should this maybe have a `#[non_exhaustive]` on it just in case?

Reply:
journald so far is Linux-only I think? And I expect Linux's definitions to stay stable.